geofencing/README.md

```diff
@@ -0,0 +1,76 @@
+# Windows Geofencing Demo with Mayhem
+
+This example demonstrates how to fuzz a C++ application that processes GPS points and checks if a car is within a defined boundary. Using Mayhem, we can identify exploitable vulnerabilities, such as buffer overflows, in the code by performing a Mayhem run.
+
+## Prerequisites
+
+To run this demo, you'll need:
+
+1. **Visual Studio 2022**: Install Visual Studio 2022 with the C++ development workload.
+2. **Mayhem CLI**: Install the Mayhem CLI from https://app.mayhem.security/-/installation.
+3. **Mayhem Authentication**: Authenticate with your Mayhem account using the following command:
+
+```
+   mayhem login https://app.mayhem.security/ <your-authentication-token>
+```
+
+## Project Setup
+
+1. **Open Developer PowerShell for Visual Studio 2022**:
+   Run the script in a Developer PowerShell prompt for Visual Studio 2022. This environment includes all necessary environment variables and tools for building the project.
+
+## Build and Run Instructions
```

Comment:
Thinking either put instructions README or ps1, not build (code duplication).

Reply:
Removed duplication from README.